docs/vsphere-golden-image.md

```diff
@@ -15,7 +15,8 @@ the patch is not available.*
 ## 2. Create the virtual machine
 
 To start with, create a new virtual machine in vSphere from the selected Windows Server distribution using the ISO image.
-See [vSphere documentation](https://docs.vmware.com/en/VMware-vSphere/index.html) for installation guidelines.
+See [vSphere documentation](https://docs.vmware.com/en/VMware-vSphere/index.html) for installation guidelines. We 
+recommend using a [virtual hardware version](https://kb.vmware.com/s/article/1003746) of `15` when creating the VM.
```

Comment:
Consider moving this change to another commit with a clear commit message. 
For example:  
```
[docs] Set hardware version in vSphere

...
```

Easier to review by docs Team.

Reply:
I don't see this as necessary

Reply:
See https://github.com/openshift/windows-machine-config-operator/pull/515/commits for references.